### PR TITLE
Feature/admin home page

### PR DESCRIPTION
**Pull request by iplavsic20**
Implemented home page for admin with three buttons for adding new user, adding new car and to see all cars.

### CODE REVIEW

File: fleetcarpooling/lib/main.dart
Comment (jsmoljanec) [inline fleetcarpooling/lib/main.dart:1]: The main.dart file is outdated.

**Review by jsmoljanec — CHANGES_REQUESTED** [review]
The main.dart file is outdated.

**Review by dsantek20 — CHANGES_REQUESTED** [review]
Fix main.dart

**Review by mtomasic2 — CHANGES_REQUESTED** [review]
It wont work like that because main.dart file is not up at latest version.

**Review by mtomasic2 — APPROVED** [review]
Nice fix

**Review by dsantek20 — APPROVED** [review]

**Review by ijelovic20 — APPROVED** [review]
Looks fine now

**Review by jsmoljanec — APPROVED** [review]

**Pull request merged**